pkg/mpc/cmd/automation.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// projectsCmd represents the automation command
+var automationCmd = &cobra.Command{
+	Use:   "automation",
+	Short: "Automation operations",
+	Long:  "Manage projects automation configs.",
+}
+
+// automationStatusCmd represents  status command
+var automationStatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Automation status",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		automationStatus, err := newClient().GetAutomationStatus(projectID)
+
+		if err != nil {
+			er(err)
```

Comment:
since `er(...)` exits, you could do the nil-check inside of that function and make the code easier to read;  also, I would rename `er()` to something which explains what it does, e.g.: `stopOnError` or something similar